core/block/editor/spaceview.go

```diff
@@ -58,21 +58,28 @@ func (s *SpaceView) Init(ctx *smartblock.InitContext) (err error) {
 func (s *SpaceView) CreationStateMigration(ctx *smartblock.InitContext) migration.Migration {
 	return migration.Migration{
 		Version: 1,
-		Proc: func(s *state.State) {
-			template.InitTemplate(s,
-				template.WithObjectTypesAndLayout([]domain.TypeKey{bundle.TypeKeySpaceView}, model.ObjectType_spaceView),
-				template.WithRelations([]domain.RelationKey{
-					bundle.RelationKeySpaceLocalStatus,
-					bundle.RelationKeySpaceRemoteStatus,
-					bundle.RelationKeyTargetSpaceId,
-				}),
-			)
-		},
+		Proc:    s.initTemplate,
 	}
 }
 
 func (s *SpaceView) StateMigrations() migration.Migrations {
-	return migration.MakeMigrations(nil)
+	return migration.MakeMigrations([]migration.Migration{
+		{
+			Version: 1,
```

Comment:
should we have version 2 here? to migrate existing objectViews

Reply:
@deff7 said no, because previous was 0